AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -112,7 +144,7 @@ - (void)_layoutNodes:(NSArray *)nodes atIndexPaths:(NSArray *)indexPaths withAni
     //TODO: There should be a fast-path that avoids all of this object creation.
     NSMutableArray *nodeBoundSizes = [[NSMutableArray alloc] initWithCapacity:kASDataControllerSizingCountPerProcessor];
     [subIndexPaths enumerateObjectsUsingBlock:^(NSIndexPath *indexPath, NSUInteger idx, BOOL *stop) {
-      ASSizeRange constrainedSize = [_dataSource dataController:self constrainedSizeForNodeAtIndexPath:indexPath];
+      ASSizeRange constrainedSize = constraintedSizeBlock(indexPath);
```

Comment:
A little worried about this, wondering if there is a better way than churning through so many block objects.